src/librustc_mir/borrow_check/mod.rs

```diff
@@ -42,7 +42,7 @@ use dataflow::DataflowResultsConsumer;
 use dataflow::FlowAtLocation;
 use dataflow::MoveDataParamEnv;
 use dataflow::{do_dataflow, DebugFormatted};
-use dataflow::{EverInitializedPlaces, MovingOutStatements};
```

Comment:
We should also be able to remove the definition of `MovingOutStatements`

Reply:
:+1:, done!